### ISSUE TITLE
DM2E Unofficial artifacts: validatedAtLevel, dm2eVersion, stability

### ISSUE THREAD

**Issue by chrmor** [DM2E/dm2e-mappings#63]
http://data.dm2e.eu/data/html/dataset/uber/dingler/20140304123936384

I see one triple like the following:

?:validatedAtLevel  ERROR (xsd:string)

is it an error?

**Comment by kba** [comment]
No, it's the unofficial DM2E namespace which has found it's way into the ingestion tool. Will move it to either omnom: or dm2e:

**Comment by kba** [comment]
I propose to replace them as such

```
dm2e_unofficial:validatedAtLevel  -> omnom:validatedAtLevel
dm2e_unofficial:stability  -> omnom:stability
dm2e_unofficial:dm2eVersion -> owl:versionInfo
```

The first two are about describing a dataset, so this is info that the publish service in omnom would generate. Otherwise we should check if prov contains equivalents.

owl:versionInfo is an annotationProperty that is rather vague in its definition so I think we could reuse that.

**Comment by edroege** [comment]
We also use owl:versionInfo for indicating the model number and revision in the owl file of the model. I guess you want to store the same information there during the ingestion, so I agree. I also think that it is a good idea to move validatedAtLevel and stability into the omnom namespace.

**Comment by kba** [comment]
OK, but let's do this later so data currently ingested remains valid.
